### PR TITLE
[TC-505] - throws 403 (rather than 404) if user is ldap-only (has no user entry …

### PR DESCRIPTION
**Pull request by mitchell852**
…in tm_user table)

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/39/

File: traffic_ops/app/conf/cdn.conf [hunk 1]
```diff
@@ -13,8 +13,9 @@
 		access_control_allow_origin => '*'
 	},
 	to => {
-		base_url   => 'http://localhost:3000',                    # this is where traffic ops app resides
-		email_from => 'no-reply@traffic-ops-domain.com'           # traffic ops email address
+		base_url				=> 'http://localhost:3000',                    																	# this is where traffic ops app resides
+		email_from				=> 'no-reply@traffic-ops-domain.com',           																# traffic ops email address
+		no_account_found_msg	=> 'A Traffic Ops user account is required for access. Please contact your Traffic Ops user administrator.'     # message to display if no TO account is found in tm_user
 	},
```
Comment (dangogh) [inline traffic_ops/app/conf/cdn.conf]: TABs added in there?

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/40/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/41/

**Pull request closed without merging**